common/src/types/v0/message_bus/volume.rs

```diff
@@ -39,18 +36,18 @@ impl Volume {
 
     /// Get the volume state.
     pub fn state(&self) -> Option<VolumeState> {
-        self.state.clone()
+        Some(self.state.clone())
     }
 
     /// Get the volume status, if any.
     pub fn status(&self) -> Option<VolumeStatus> {
-        self.state.as_ref().map(|s| s.status.clone())
+        Some(self.state.status.clone())
     }
 }
 
 impl From<Volume> for models::Volume {
     fn from(volume: Volume) -> Self {
-        models::Volume::new_all(volume.spec(), volume.state().into_opt())
+        models::Volume::new_all(volume.spec(), volume.state)
```

Comment:
Shouldn't we have the same behaviour here: either move both members out of target volume or use getters ?

Reply:
Yes that should be state()

Reply:
Done